app/App/Account/Account/Balances/helpers.js

```diff
@@ -36,3 +36,14 @@ export function balance (rawBalance, quote = {}) {
     displayValue: totalValue.isZero() ? '0' : formatUsdRate(totalValue, 0)
   }
 }
+
+export const sortByTotalValue = (a, b) => {
+  if (a.totalValue.plus(b.totalValue).gt(0)) {
```

Comment:
what is this block checking for?

Reply:
The first block is checking if we need to sort by totalValue rather than token amounts

Reply:
kinda thought so but I don't follow what this guard is checking for?

would something like this make sense:
``
let totalValueDiff = b.totalValue.minus(a.totalValue)
if (!totalValueDiff.isZero()) return totalValueDiff.toNumber()
```

Reply:
ahh I see, it's a check for "is one of these values positive". this check actually isn't even correct because if the total values are the same the sort will return zero when it should instead try to do the secondary sort by the balance. this would make another good unit test case

Reply:
Whilst unit testing I came to the same conclusion as your comment above ;)